BlueprintUICommonControls/Tests/Sources/LabelTests.swift

```diff
@@ -108,7 +108,7 @@ class LabelTests: XCTestCase {
     }
 
     fileprivate func compareSnapshot(identifier: String? = nil, file: StaticString = #file, testName: String = #function, line: UInt = #line, configuration: (inout Label) -> Void) {
-        var label = Label(text: "Hello, world. This is a long run of text that should wrap at some point. Someone should improve this test by adding a joke or something. Alright, it's been fun!")
+        var label = Label(text: "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.")
```

Comment:
Intentional?

Reply:
yeah was testing snapshot tests to make sure they built

Reply:
hence the `DRAFT` ;)

Reply:
![hrYQS5h](https://user-images.githubusercontent.com/327847/103714496-a0a60400-4f73-11eb-9675-2e9d19a6a3b1.gif)